packages/medusa/src/models/payment-session.ts

```diff
@@ -22,6 +22,7 @@ export enum PaymentSessionStatus {
 }
 
 @Unique("OneSelected", ["cart_id", "is_selected"])
+@Unique("UniqPaymentSessionCartIdProviderId", ["cart_id", "provider_id"])
```

Comment:
praise: Nice!

Reply:
thanks @srindom with who we come up with that solution 🚀